README.md

```diff
@@ -114,6 +114,7 @@ The `--fix` option on the command line automatically fixes problems reported by
 |  | [classic-decorator-no-classic-methods](./docs/rules/classic-decorator-no-classic-methods.md) | Prevent usage of classic APIs such as get/set in classes that aren't explicitly decorated with @classic |
 |  | [computed-property-getters](./docs/rules/computed-property-getters.md) | Enforce the consistent use of getters in computed properties |
 |  | [no-proxies](./docs/rules/no-proxies.md) | Disallows using array or object proxies |
+|  | [no-actions-hash](./docs/rules/no-actions-hash.md) | Disallows the actions hash in components, controllers and routes |
```

Comment:
Be sure to use `yarn update` to generate this properly. It's out-of-order right now.